src/coreclr/inc/corinfo.h

```diff
@@ -3436,7 +3436,9 @@ class ICorDynamicInfo : public ICorStaticInfo
 //
 // RISCV64 relocation types
 //
-#define IMAGE_REL_RISCV64_PC            0x0003
+#define IMAGE_REL_RISCV64_CALL_PLT      0x0003  // auipc + jalr
```

Comment:
> CALL

Do we expect that this is never going to be used with branch instruction (e.g. for tailcalls)?

Reply:
I have not realized that this name was inspired by R_RISCV_CALL_PLT. It is fine then.